### ISSUE TITLE
how to deploy jars

### ISSUE THREAD

**Issue by ariovistus** [Open-RIO/ToastAPI#20]
java ecosystem outsider here.

We have a local jar file - the one for the navx not (to my knowledge) in any maven repository. How would you configure your build.gradle to ensure the deploy process includes that jar file?

**Comment by JaciBrunning** [comment]
For jar file dependencies, create a new folder called `libs` and put your NavX jar in there. Make your build.gradle `dependencies {}` block resemble the following.

```
dependencies {
    compile fileTree(dir: 'libs/', include: '*.jar')
}
```

This will allow your IDE and Compiler to link against the NavX jar. If you want to deploy the NavX jar to the roborio, you can do the following:

```
dependencies {
    compile fileTree(dir: 'libs/', include: '*.jar')
    toastLibrary fileTree(dir: 'libs/', include: '*.jar')
}
```

**Comment by ariovistus** [comment]
that seems to work, except it deposits the jar file on the roborio with the name 'unspecified.jar'.
How about simulation? the jar file doesn't seem to get copied there

**Comment by JaciBrunning** [comment]
I'll reopen with the GradleRIO label. I'll see what I can do. Are you using gradle  simulation or IDE simulation?

**Comment by ariovistus** [comment]
gradle

**Comment by JaciBrunning** [comment]
Fixed in `07ab890` on [GradleRIO](https://github.com/Open-RIO/GradleRIO/commit/07ab890fb893d92f815c2c945690b45c300198cd)
